db/seeds.rb

```diff
@@ -7,12 +7,23 @@
 end
 
 [
-  ["A Clockwork Orange", "0393312836", "db/sample_data/a-clockwork-orange.jpg"],
-  ["Crime and Punishment", "0679734503", "db/sample_data/crime-and-punishment.jpg"],
-  ["Transcending CSS", "0321410971", "db/sample_data/transcending-css.jpg"],
-  ["The Very Hungry Caterpillar", "0399226907", "db/sample_data/the-very-hungry-caterpillar.jpg"]
-].each do |name, isbn, cover|
+  "Dystopian Fiction",
+  "Psychological Fiction",
+  "Reference",
+  "Children's Literature"
+].each do |name|
+  genre = Genre.find_or_create_by(name: name)
+end
+
+[
+  ["A Clockwork Orange", "0393312836", "db/sample_data/a-clockwork-orange.jpg", "Dystopian Fiction"],
+  ["Crime and Punishment", "0679734503", "db/sample_data/crime-and-punishment.jpg", "Psychological Fiction"],
+  ["Transcending CSS", "0321410971", "db/sample_data/transcending-css.jpg", "Reference"],
+  ["The Very Hungry Caterpillar", "0399226907", "db/sample_data/the-very-hungry-caterpillar.jpg", "Children's Literature"]
+].each do |name, isbn, cover, genre|
+  genre = Genre.find_by(name: genre)
```

Comment:
Let's solve a minor inefficiency and have some fun with ruby.

As this collection gets larger it is going to be less and less effective to find the genre each iteration.

Can you think of a way to not have to find the genre each loop? Keep in mind we are creating them in the loop above starting on line 10.